src/AzureClient/AzureClient.cs

```diff
@@ -543,7 +567,10 @@ public async Task<ExecutionResult> GetJobResultAsync(IChannel? channel, string j
                 //                 cancellation token support.
                 var request = WebRequest.Create(job.OutputDataUri);
                 using var responseStream = (await request.GetResponseAsync()).GetResponseStream();
-                return responseStream.ToHistogram(Logger).ToExecutionResult();
+                return responseStream.ToHistogram(
```

Comment:
Yes, @bettinaheim said that she wants a histogram with a single record for the return value of the entry point as the output of a full-state simulator job.

Reply:
I think that would be a pretty surprising and counter-intuitive user experience for Q# notebook users especially, in that running an operation locally and via the full-state simulator would then give very different visualizations as a result.

Reply:
I couldn't agree more, but we've decided for now to go forward with it like this. Perhaps before merging into main we can revisit this topic.

Reply:
Since how to handle the output will be addressed in another PR and the conversation on how to do that is still ongoing, we could leave this part as is for now (since it is going into a feature branch) with the expectation that it will be addressed before any of these changes are merged to main.

Reply:
Agreed; this should be resolved before the final merge to main, but let's defer that for now.